lib/govuk_index/publishing_event_worker.rb

```diff
@@ -47,14 +47,17 @@ def process_action(actions, routing_key, payload)
       )
       presenter.valid!
 
+      identifier = "#{presenter.base_path} #{presenter.type || "'unmapped type'"}"
```

Comment:
Can it not be `|| 'unmapped type'}"`?

Or do you want the inverted commas for extra noticability?

Reply:
the second one, but happy to remove..

Reply:
shruggy mcshrugface